src/main/java/me/zeroeightsix/kami/util/EntityUtils.kt

```diff
@@ -237,7 +238,7 @@ object EntityUtils {
             if (entity == mc.player) continue
             if (entity is EntityPlayer) {
                 if (!player[0]) continue
-                if (!player[1] && Friends.isFriend(entity.name)) continue
+                if ((!player[1] || KamiMod.MODULE_MANAGER.isModuleEnabled(AntiFriendHit::class.java)) && Friends.isFriend(entity.name)) continue
```

Comment:
Why should disabling antifriendhit suddenly break all friend related modules??

Reply:
Does it? This is an or statement (which if anything should break it when AntiFriendHit is enabled not disabled) so all modules targeting friends should not be able to since you have AntiFriendHit enabled, which should prevent you from targeting your friends. I'll go and test all modules that use `EntityUtils::getTargetList` and see if any do not work as intended.

Reply:
This change shouldn't have been made in the first place, there's no reason to change this.

Reply:
Before my changes, when you have aura enabled with friends enabled and AntiFriendHit your character swings at friends without doing damage or actually hitting them, therefore causing you to possibly target the wrong person, wasting an attack on a friend that could have hit your enemy.